files/ko/web/javascript/reference/global_objects/null/index.html

```diff
@@ -3,22 +3,31 @@
 slug: Web/JavaScript/Reference/Global_Objects/null
 tags:
   - JavaScript
+  - Language feature
   - Literal
   - Primitive
-  - Reference
-translation_of: Web/JavaScript/Reference/Global_Objects/null
+browser-compat: javascript.builtins.null
+
 ---
-<div>{{jsSidebar("Objects")}}</div>
+<div>{{jsSidebar("Operators")}}</div>
 
-<p><strong><code>null</code></strong>은 JavaScript의 {{Glossary("Primitive", "원시 값")}} 중 하나로, 어떤 값이 의도적으로 비어있음을 표현하며 불리언 연산에서는 {{glossary("falsy", "거짓")}}으로 취급합니다.</p>
+<p>
+  <strong><code>null</code></strong>은 JavaScript의 <a href="/ko/docs/Glossary/Primitive">원시 값</a> 중 하나로, 어떤 값이 의도적으로 비어있음을 표현하며 불리언
+   연산에서는 <a href="/ko/docs/Glossary/Falsy">거짓</a>으로 취급합니다.
+</p>
 
 <div>{{EmbedInteractiveExample("pages/js/globalprops-null.html")}}</div>
 
+<h2 id="Syntax">구문</h2>
 
+<pre class="brush: js">null</pre>
 
-<h2 id="설명">설명</h2>
+<h2 id="Description">설명</h2>
 
-<p><code>null</code>은 리터럴로서, <code>null</code>로 작성할 수 있습니다. <code>null</code>은 {{jsxref("undefined")}}와 같은 전역 객체의 식별자가 아닙니다. 대신 식별되지 않은 상태를 나타내며 해당 변수가 어떠한 객체도 가리키고 있지 않음을 표시합니다. API에서는 대개 객체를 기대했지만, 어떤 적합한 객체도 존재하지 않을 때 반환합니다.</p>
+<p><code>null</code>은 문자 그대로 <code>null</code>이라 씁니다. <code>null</code>은 
+  {{jsxref("Global_Objects/undefined","undefined")}}과 같이 글로벌 객체의 속성에 대한 식별자가 아닙니다. 대신 <code>null</code>은 식별이 부족이나 변수가 
+  객체를 가리키지 않음을 표현합니다. API에서 <code>null</code>은 종종 객체가 있어야하지만 연관된 개체가 없는다는 항목을 받아올 때 사용합니다.
```

Comment:
"있어야하지만" -> "있어야 하지만"
"개체" -> "객체"

> In APIs, null is often retrieved in a place where an object can be expected but no object is relevant. 

에서, "In APIs"가 주체라는 것이 아니고 API 응답을 말하는 것이므로 "is ... retrieved"는 "API에서 null은 ... 받아올 때 사용합니다"보단 "API에서는 null을 ... 사용합니다"가 더 자연스럽게 읽힙니다.

> API에서는 종종 `null`을 객체가 있어야 하는 항목에 연관된 객체가 없을 때 사용합니다.

그런데 원문이 "must exists"가 아니고 "can be expected"네요. 그러면 "있어야 하는" 이라는 말은 너무 강합니다.

```suggestion
  객체를 가리키지 않음을 표현합니다. API에서는 <code>null</code>을 종종 관련된 객체가 존재하지 않을 때 그 객체 대신 사용합니다.
```